Documentation/PageTsconfig/TceMain.rst

```diff
@@ -481,17 +481,37 @@ translateToMessage
 
 :aspect:`Datatype`
     string
+    
+:aspect:`Default`
+    Translate to %s:
 
 :aspect:`Description`
-    Defines the string that will be prepended to every field value if you copy an element to another
-    language version. The special string "%s" will be replaced with the language title.
+    Defines the string that will be prepended to some field values if you copy an element to another
+    language version. This applies to all fields where the TCA columns property 
+    :ref:`l10n_mode <t3tca:columns-properties-l10n-mode>` is set to :php:`prefixLangTitle`.
+    
+    The special string "%s" will be replaced with the language title.
+    
+    You can globally disable the prepending of the string by setting `translateToMessage` to
+    an empty string. You can disable the message to a certain field by setting the `l10n_mode`
```

Comment:
What does the `l10n_mode` part refer to? If [l10n_mode](https://docs.typo3.org/m/typo3/reference-tca/11.5/en-us/Columns/Properties/L10nMode.html) then I don't think this is valid since that option only accepts 2 values, an empty string is not valid here.

Reply:
Please have a look at this PR? Should we rather use unset? setting it to an empty strings works.
**https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/pull/519**